### DISCUSSION TITLE
데이터 이야기

### DISCUSSION THREAD

**Discussion by cyberjam** [LifeIsAlone/public-home#53] [General]
<img width="699" alt="image" src="https://user-images.githubusercontent.com/20448972/192770725-3dcd9708-56a6-4061-8ca2-7561a67d5cea.png">
이번에는 스프레드 시트로 연동하여 데이터 처리가 편했습니다. 경기, 인천지역도 추가하니 보기가 좋군요. 나중에는 전국을 대상으로 띄워 보고 싶네요. 
아직까진 첫 로딩시 조금 시간이 걸리긴하는데 리액트보다는 빠릅니다.
https://public-home.vercel.app/

그리고 데이터 연동이 되었다하더라도 여전히 각 공고마다 다른 컬럼이름으로 수작업을 하는 부분이 남아 있습니다. 관련 공공데이터api를 찾아봐야겠네요.

**Comment by sangjuun-park** [comment]
이런 데이터를 보니.. 저도 행복주택 공고를 찾아봐야 ,.,...으흠 으흠 고생많으세요!

**Reply by cyberjam** [reply]
나중에 같이 보시죠 :)

**Reply by sangjuun-park** [reply]
네 부탁드리겠습니다~~ (●'◡'●)
